brigade-worker/src/job.ts

```diff
@@ -150,6 +150,8 @@ export class JobResourceRequest {
  * the Docker container to be run.
  * */
 export abstract class Job {
+  public static readonly MAX_JOB_NAME_LENGTH = 36;
```

Comment:
Any particular reason this is not an exported constant?

Otherwise, this LGTM 👍

Reply:
As the class is getting exported and this value very much relates to Job (than being independent). The value can always be accessed outside the class (being public) like `Job.MAX_JOB_NAME_LENGTH`